templates/database.yml.j2

```diff
@@ -0,0 +1,42 @@
+#
+# PRODUCTION
+#
+production:
+  adapter: postgresql
+  encoding: unicode
+  database: {{gitlab_db_name}}
+  pool: {{gitlab_db_pool}}
```

Comment:
spaces...spaces everywhere

Reply:
?

Reply:
spaces missing between the curly braces and the expression inbetween

Reply:
Yep, please fix this so I can start reviewing

Reply:
done

Reply:
done